cmd/load-smoke-data/load-smoke-data.go

```diff
@@ -58,6 +60,9 @@ func main() {
 	db := client.Database(dbName)
 	grip.EmergencyFatal(db.Drop(ctx))
 
+	logsDb := client.Database(logsDbName)
+	grip.EmergencyFatal(logsDb.Drop(ctx))
```

Comment:
Let's avoid dropping this for the same reason that we don't want to do it in the integration test. could you instead remove the documents that you'll be inserting?